paymentapi/src/main/resources/application.properties

```diff
@@ -20,3 +20,6 @@ payment_default_return_url=http://localhost:8080/
 payment_default_notify_url=http://localhost:8080/
 payment_default_submerchant_id=36240
 payment_default_cp=PRO-31312-1
+
+# namespaces separated by comma
+enabled_namespaces.payment_method_filter.ORDER_TYPE=asukaspysakointi
```

Comment:
Should this config come from configuration settings and not from environment variable? These could be then easier managed, no need to new release every time values needs to be changes?